napytau/gui/components/graph.py

```diff
@@ -42,22 +45,27 @@ def plot(self, value: int, appearance: str) -> Canvas:
         :param value: The value.
         :return: The canvas.
         """
+
+        # Placeholder datapoints:
+        y_data = [2, 4, 5, 7, 8, 9, 11, 15]
+        distances = [1, 2, 3, 4, 5, 6, 7, 8]
+        x_data = [0, 1, 2, 3, 4, 5, 6, 7]
+
         # the figure that will contain the plot
         fig = Figure(figsize=(3, 2), dpi=100, facecolor="white", edgecolor="black")
 
         # setting color in dependence of appearance mode
         if appearance == "Light":
             main_color = "white"
             secondary_color = "#000000"
+            main_marker_color = "g"
```

Comment:
I would suggest either adding a comment, or even better a constant with a descriptive name to describe what color this translates to (i assume green)